### PR TITLE
Implementatio for starter.

### PR DESCRIPTION
**Pull request by caalador**
Implementation for the Quarkus starter

### CODE REVIEW

File: src/main/java/com/example/starter/base/TestServlet.java [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+import javax.servlet.annotation.WebInitParam;
+import javax.servlet.annotation.WebServlet;
+
+import com.vaadin.quarkus.QuarkusVaadinServlet;
+
+@WebServlet(urlPatterns = "/*", name = "TestServlet", asyncSupported = true, initParams = {
+        @WebInitParam(name = "org.atmosphere.websocket.suppressJSR356", value = "true") })
+public class TestServlet extends QuarkusVaadinServlet{
+    // This servlet is here to give the suppressJSR356 parameter
```
Comment (pleku) [inline src/main/java/com/example/starter/base/TestServlet.java:11]:
```suggestion
    // This servlet is here to give the suppressJSR356 parameter
    // Needed until https://github.com/vaadin/quarkus/issues/5 is fixed
```

**Review by pleku — CHANGES_REQUESTED** [review]
Also got an error when running the app, described in slack.

Testing the native compilation next

File: pom.xml [hunk 2]
```diff
@@ -0,0 +1,233 @@
...
+                            <goal>build</goal>
+                            <goal>generate-code</goal>
+                            <goal>generate-code-tests</goal>
+                        </goals>
+                    </execution>
+                </executions>
+            </plugin>
+            <plugin>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <version>3.8.1</version>
```
[the remark this answers is not in the document]
Comment (caalador) [reply pom.xml:145]: Quarkus has a minimum requirement of `Apache Maven 3.6.2+`
I guess that is from the archetype, but it shouldn't be needed specifically if the maven version is high enough

File: pom.xml [hunk 3]
```diff
@@ -0,0 +1,233 @@
...
+
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-undertow-websockets</artifactId>
+        </dependency>
+
+        <dependency>
+            <groupId>com.vaadin</groupId>
+            <artifactId>vaadin-core-jandex</artifactId>
+            <version>${vaadin.version}</version> <!-- TODO: To be added to the vaadin-core BOM -->
```
[the remark this answers is not in the document]
Comment (caalador) [reply pom.xml:103]: Waiting for Platform build to pass.

File: pom.xml (hunk 2 above)
[the remark this answers is not in the document]
Comment (caalador) [reply pom.xml]: done

File: pom.xml (hunk 3 above)
[the remark this answers is not in the document]
Comment (caalador) [reply pom.xml]: done.

**Review by pleku — APPROVED** [review]

**Pull request merged**